lib/ansible/modules/network/avi/avi_actiongroupconfig.py

```diff
@@ -107,32 +84,43 @@
         type: string
     name:
```

Comment:
The examples show `name:` is needed, if that's correct then this needs documenting